cypress/support/e2e.ts

```diff
@@ -1 +1,13 @@
 import './commands';
+
+const userNameInputSelector = '[type="email"]';
+const passwordInputSelector = '[type="password"]';
+
+Cypress.Commands.add('login', () => {
+  cy.visit('/login');
+  cy.get('form').within(() => {
+    cy.get(userNameInputSelector).type('you@email.com');
```

Comment:
For this part u can use [cypress-data-session](https://github.com/bahmutov/cypress-data-session) plugin.

Reply:
added